### PR TITLE
Restrained spacepod passenger can now exit pod after 2 minutes

### PR DESCRIPTION
**Pull request by MINIMAN10000**
Had someone on discord complaining about spending an entire round stuck in a spacepod as a traitor because the space pod pilot have shoved them in the pod and ran off and died. This left her stuck in the spacepod for the rest of the round and admins refused to do anything about it.

Being stuck inside of a spacepod the whole round because a spacepod pilot dying is just something that shouldn't be able to happen.

Thus this change makes it so a restrained ( handcuffed ) person can stumble their way out of the securitypod after 2 minutes if it doesn't move for 2 minutes.

For reference it actually currently takes 1 minute to get out of a cryotube ( even though it says 2 minutes it's 600 deciseconds which is 60 seconds )

:cl:
Tweak: Restrained spacepod passanger can now exit pending a 2 minute wait time without moving.
/:cl:

### CODE REVIEW

File: code/modules/spacepods/spacepod.dm [hunk 1]
```diff
@@ -812,11 +812,17 @@ obj/spacepod/proc/add_equipment(mob/user, var/obj/item/spacepod_equipment/SPE, v
...
-	if(usr.incapacitated()) // unconscious and restrained people can't let themselves out
+	if(usr.stat) // unconscious people can't let themselves out
 		return
 
 	occupant_sanity_check()
 
+	if(usr.restrained())
+		to_chat(usr, "<span class='notice'>You attempt to stumble out of the [src]. This will take two minutes.</span>")
+		if(do_after(usr, 1200, target = src))
+		else
```
Comment (Crazylemon64) [inline code/modules/spacepods/spacepod.dm]:
```
if(!do_after(...))
   return
```

File: code/modules/spacepods/spacepod.dm [hunk 2]
```diff
@@ -812,11 +812,17 @@ obj/spacepod/proc/add_equipment(mob/user, var/obj/item/spacepod_equipment/SPE, v
 	if(!istype(user))
 		return
 
-	if(usr.incapacitated()) // unconscious and restrained people can't let themselves out
+	if(usr.stat) // unconscious people can't let themselves out
```
Comment (Crazylemon64) [inline code/modules/spacepods/spacepod.dm]: `usr.stat != CONSCIOUS`

**Comment by alexkar598** [comment]
I would say make it 5 minutes,lorewise: its a complex space pod made to resists space so it takes time to brute force your way out and gameplay wise: it can take more than 2 minutes get back to the security pod bay if you,let's say catch a traitor attempting to destoy the AI

File: code/modules/spacepods/spacepod.dm [hunk 3]
```diff
@@ -812,11 +812,17 @@ obj/spacepod/proc/add_equipment(mob/user, var/obj/item/spacepod_equipment/SPE, v
...
 		return
 
 	occupant_sanity_check()
 
+	if(usr.restrained())
+		to_chat(usr, "<span class='notice'>You attempt to stumble out of the [src]. This will take two minutes.</span>")
```
Comment (Citinited) [inline code/modules/spacepods/spacepod.dm]: This should give some message to the pilot at least, so they know if their prisoner is trying to break out

**Comment by MINIMAN10000** [comment]
@alexkar598 it shouldn't matter how long it takes to get from A to B, so long as the pod moves it will interrupt his escape.

**Comment by alexkar598** [comment]
Ahh,tought it did not reset it due to being inside a container

**Pull request merged**